### PR TITLE
EVA-934: dbSNP import progress status view

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

File: src/js/dbSNP-import/dbSNP-import-progress.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            '<div><table id="dbSNP-import-table" class="responsive-table hover tablesorter"><thead>' +
+            '<tr>' +
+            '<th>Common name</th>' +
+            '<th>Scientific name</th>' +
+            '<th>Taxonomy ID</th>' +
+            '<th>INSDC assembly accession</th>' +
+            '<th>dbSNP build</th>' +
+            '<th>Supported by Ensembl</th>' +
+            '<th>Suitable for Variant Browser</th>' +
+            '<th>All variants match<br>INSDC Assembly</th>' +
```
Comment (cyenyxe) [inline src/js/dbSNP-import/dbSNP-import-progress.js]: INSDC **a**ssembly

File: src/index.html [hunk 2]
```diff
@@ -1016,7 +1033,7 @@ <h2 id="variantInfo"  data-magellan-target="variantInfo" class="padding-bottom-l
             if (parArr) {
                 for (var i = 0; i < parArr.length; i++) {
                     parr = parArr[i].split("=");
-                    data.id = parr[0];
+                    data.id = parr[0].replace(/-/g, " ");
```
Comment (cyenyxe) [inline src/index.html]: Does this translate the URLs with dashes (such as `Variant-Browser`) to URLs with white spaces (`Variant Browser`)?

File: tests/acceptance/dbSNP_import.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        driver.findElement(By.xpath("//li//a[text()='dbSNP Import Progress']")).click();
+    });
+
+    test.after(function() {
+        config.shutdownDriver(driver);
+    });
+
+    test.describe('dbSNP Import Progress table check', function() {
+
+        test.it('Check "Species" column should not be empty', function() {
```
Comment (cyenyxe) [inline tests/acceptance/dbSNP_import.js]: "Scientific name" instead of "Species".

File: tests/acceptance/dbSNP_import.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+                    for (var i = 0; i < rows.length; i++){
+                        rows[i].getAttribute("innerHTML").then(function(text){
+                            chai.assert.notEqual(text, null);
+                        });
+                    }
+                });
+            });
+        });
+
+        test.it('Check "To variant browser" column should not be empty', function() {
```
Comment (cyenyxe) [inline tests/acceptance/dbSNP_import.js]: Suuitable for Variant Browser

File: tests/acceptance/dbSNP_import.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+                    for (var i = 0; i < rows.length; i++){
+                        rows[i].getAttribute("innerHTML").then(function(text){
+                            chai.assert.notEqual(text, null);
+                        });
+                    }
+                });
+            });
+        });
+
+        test.it('Check "Variants mapped to assembly" column should not be empty', function() {
```
Comment (cyenyxe) [inline tests/acceptance/dbSNP_import.js]: All variants match INSDC assembly

File: tests/acceptance/dbSNP_import.js [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+                    }
+                });
+            });
+        });
+
+    });
+
+});
+
+
```
Comment (cyenyxe) [inline tests/acceptance/dbSNP_import.js]: Remove unnecessary empty lines.

File: src/index.html (hunk 2 above)
Comment (kandaj) [reply src/index.html]: No in the URLs you'll see as Variant-Browser

Comment (kandaj) [reply src/index.html]: other changes are updated

**Pull request merged**